app/models/content.js

```diff
@@ -25,5 +25,5 @@ export default DS.Model.extend({
     }
   }),
   progress: DS.belongsTo('progress'),
-  //section: DS.belongsTo('section')
+  section: DS.belongsTo('section')
```

Comment:
This shall remain commented for the simple reason, a content may belong to many sections at a time.